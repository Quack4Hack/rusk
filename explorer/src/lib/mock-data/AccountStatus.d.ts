type AccountStatus = {
  balance: bigint;
  nonce: bigint;
};
