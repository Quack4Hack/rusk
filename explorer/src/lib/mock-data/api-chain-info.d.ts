type APIBlockHeader = {
  __typename: string;
  feespaid: number;
  hash: string;
  height: number;
  nextblockhash?: string;
  prevblockhash?: string;
  reward: number;
  seed: string;
  statehash?: string;
  timestamp: string;
  ts: number;
  version: string;
};

type APIBlockTransactions = {
  data: APITransaction[];
  stats: {
    averageGasPrice: number;
    gasLimit: number;
    gasUsed: number;
  };
};

type APIBlock = {
  __typename: string;
  header: APIBlockHeader;
  transactions: APIBlockTransactions;
};

type APIChainInfo = {
  blocks: APIBlock[];
  transactions: APITransaction[];
};

type APISearchBlockResult = {
  data: {
    data: {
      blocks: {
        header: { hash: string };
      }[];
    };
  };
};

type APISearchNoResult = {
  data: {};
};

type APISearchTransactionResult = {
  data: {
    data: {
      transactions: {
        __typename: string;
        tx: { id: string };
        txid: string;
      }[];
    };
  };
};

type APISearchResult =
  | APISearchBlockResult
  | APISearchNoResult
  | APISearchTransactionResult;

type APITransaction = {
  __typename: string;
  blockhash: string;
  blockheight: number;
  blocktimestamp: string;
  blockts: number;
  contract: string;
  feepaid: number;
  gaslimit: number;
  gasprice: number;
  gasspent: number;
  method?: string;
  success: boolean;
  txerror: string;
  txid: string;
  txtype: string;
};
