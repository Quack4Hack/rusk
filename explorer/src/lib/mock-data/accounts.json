[
  {
    "address": "0xe5601aae15c846ead69dab4c86a049405",
    "balance": 90993,
    "staked_balance": 13802,
    "active_stake": 9795,
    "inactive_stake": 4007,
    "rewards": 987
  },
  {
    "address": "0xea8bdd8e92be744e843c644bf581455ea",
    "balance": 83097,
    "staked_balance": 13179,
    "active_stake": 4469,
    "inactive_stake": 8710,
    "rewards": 3561
  },
  {
    "address": "0x883c4c22510df65fe29ebf6dd0ed710f6",
    "balance": 93610,
    "staked_balance": 49767,
    "active_stake": 39775,
    "inactive_stake": 9992,
    "rewards": 3616
  },
  {
    "address": "0x39aae05835f723cc8a7152b63f7d73ffc",
    "balance": 12804,
    "staked_balance": 5582,
    "active_stake": 4737,
    "inactive_stake": 845,
    "rewards": 4834
  },
  {
    "address": "0xad36bbf36e1cca4b3997abf14f611c2de",
    "balance": 92044,
    "staked_balance": 9446,
    "active_stake": 2944,
    "inactive_stake": 6502,
    "rewards": 1766
  },
  {
    "address": "0xdd211bb967327a5edd05592c5e44816e8",
    "balance": 84539,
    "staked_balance": 79912,
    "active_stake": 37637,
    "inactive_stake": 42275,
    "rewards": 2528
  },
  {
    "address": "0x1a229291051ec42a3e8abcfb7e9400cb0f",
    "balance": 61410,
    "staked_balance": 1460,
    "active_stake": 1324,
    "inactive_stake": 136,
    "rewards": 1399
  },
  {
    "address": "0xdc7f07a38332a73b6d3eb456c57548509",
    "balance": 35094,
    "staked_balance": 7814,
    "active_stake": 4616,
    "inactive_stake": 3198,
    "rewards": 972
  },
  {
    "address": "0x156a5e0592cfba9e2d6c9fe95d6f6bbbed",
    "balance": 85970,
    "staked_balance": 44119,
    "active_stake": 12865,
    "inactive_stake": 31254,
    "rewards": 4555
  },
  {
    "address": "0x127de00849726e14bb7e889f42fb1cc4d5",
    "balance": 32429,
    "staked_balance": 26072,
    "active_stake": 18899,
    "inactive_stake": 7173,
    "rewards": 849
  }
]
