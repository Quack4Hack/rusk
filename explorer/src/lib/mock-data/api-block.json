{
  "data": {
    "blocks": [
      {
        "__typename": "Block",
        "header": {
          "__typename": "Header",
          "feespaid": 580718,
          "hash": "bd5c99bb720b03500e89f103fe66113ba62f2e124ed9651563f38fd15977719f",
          "prevblockhash": "07b74b35c2c7cf8f41426cd0870bafa1a2c7adee3fdd876643548096186fc4cb",
          "nextblockhash": "6011556208a85e6001bd01ccbf936486b91318a7f6cbcf7ab810adf6fae34204",
          "reward": 16000000000,
          "seed": "af15447e3a004a79d4ae8b084f7b76b78d95880bb63e1cfa79250a310731f52e6d84ee42a5d6fc2cb99c5b1f489761f6",
          "statehash": "20bb0a677b93f084afadfd34bec3ac3feee33a020b81d9549afa2268e8543acb",
          "height": 495868,
          "ts": 1713298022,
          "timestamp": "2024-04-16 20:07:02 +0000 UTC",
          "version": "0"
        },
        "transactions": {
          "data": [
            {
              "__typename": "SpentTransaction",
              "blockhash": "bd5c99bb720b03500e89f103fe66113ba62f2e124ed9651563f38fd15977719f",
              "blockheight": 495868,
              "feepaid": 290866,
              "gasspent": 290866,
              "txerror": "",
              "success": true,
              "blockts": 1713298022,
              "blocktimestamp": "2024-04-16 20:07:02 +0000 UTC",
              "gaslimit": 500000000,
              "gasprice": 1,
              "txid": "3a3f6f90a1012ae751b4448bcb8e98def0ba2b18170239bd69fcf8e2e37f0602",
              "txtype": "1",
              "contract": "Transfer",
              "method": "transfer"
            },
            {
              "__typename": "SpentTransaction",
              "blockhash": "bd5c99bb720b03500e89f103fe66113ba62f2e124ed9651563f38fd15977719f",
              "blockheight": 495868,
              "feepaid": 289852,
              "gasspent": 289852,
              "txerror": "",
              "success": true,
              "blockts": 1713298022,
              "blocktimestamp": "2024-04-16 20:07:02 +0000 UTC",
              "gaslimit": 500000000,
              "gasprice": 1,
              "txid": "07bfabea1d94c16f2dc3697fa642f6cecea6e81bf76b9644efbb6e2723b76d00",
              "txtype": "1",
              "contract": "Transfer",
              "method": "transfer"
            }
          ],
          "stats": {
            "gasUsed": 580718,
            "averageGasPrice": 1,
            "gasLimit": 5000000000
          }
        }
      }
    ]
  }
}
