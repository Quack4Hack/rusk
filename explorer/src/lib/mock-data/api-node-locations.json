{
  "data": [
    {
      "lat": 52.352,
      "lon": 4.9392
    },
    {
      "lat": 31.2222,
      "lon": 121.4581
    },
    {
      "lat": 51.5074,
      "lon": -0.1196
    },
    {
      "lat": 49.405,
      "lon": 11.1617
    },
    {
      "lat": 60.1719,
      "lon": 24.9347
    },
    {
      "lat": 52.6959,
      "lon": 6.1847
    },
    {
      "lat": 33.5939,
      "lon": -112.303
    },
    {
      "lat": 48.8566,
      "lon": 2.35222
    },
    {
      "lat": 33.5939,
      "lon": -112.303
    },
    {
      "lat": 1.32123,
      "lon": 103.695
    },
    {
      "lat": 27.9822,
      "lon": -81.6706
    },
    {
      "lat": 41.8879,
      "lon": -88.1995
    },
    {
      "lat": 35.6092,
      "lon": 139.73
    },
    {
      "lat": 41.8879,
      "lon": -88.1995
    },
    {
      "lat": 47.6034,
      "lon": -122.3414
    },
    {
      "lat": 39.0438,
      "lon": -77.4874
    },
    {
      "lat": 47.6348,
      "lon": -122.3451
    },
    {
      "lat": 49.405,
      "lon": 11.1617
    },
    {
      "lat": 21.0467,
      "lon": 105.748
    },
    {
      "lat": 47.6348,
      "lon": -122.3451
    },
    {
      "lat": 43.709,
      "lon": -79.4057
    },
    {
      "lat": 47.6034,
      "lon": -122.3414
    },
    {
      "lat": 47.6034,
      "lon": -122.3414
    },
    {
      "lat": 47.6034,
      "lon": -122.3414
    },
    {
      "lat": 42.8856,
      "lon": -78.8736
    },
    {
      "lat": 1.35208,
      "lon": 103.82
    },
    {
      "lat": 40.8364,
      "lon": -74.1403
    },
    {
      "lat": 49.405,
      "lon": 11.1617
    },
    {
      "lat": 49.405,
      "lon": 11.1617
    },
    {
      "lat": 1.28967,
      "lon": 103.85
    },
    {
      "lat": 60.1797,
      "lon": 24.9344
    },
    {
      "lat": 21.0278,
      "lon": 105.834
    },
    {
      "lat": 35.6092,
      "lon": 139.73
    },
    {
      "lat": 38.6287,
      "lon": -90.1988
    },
    {
      "lat": 37.3931,
      "lon": -121.962
    },
    {
      "lat": 50.475,
      "lon": 12.365
    },
    {
      "lat": 50.1169,
      "lon": 8.6837
    },
    {
      "lat": 45.3151,
      "lon": -73.8779
    },
    {
      "lat": 37.3931,
      "lon": -121.962
    },
    {
      "lat": 1.32123,
      "lon": 103.695
    },
    {
      "lat": 40.793,
      "lon": -74.0247
    },
    {
      "lat": 20.9577,
      "lon": 105.783
    },
    {
      "lat": 49.405,
      "lon": 11.1617
    },
    {
      "lat": 41.8874,
      "lon": -87.6318
    },
    {
      "lat": -33.8715,
      "lon": 151.2006
    },
    {
      "lat": 41.8879,
      "lon": -88.1995
    },
    {
      "lat": 30.3298,
      "lon": -81.6492
    },
    {
      "lat": -33.8715,
      "lon": 151.2006
    },
    {
      "lat": 50.7988,
      "lon": -1.0944
    },
    {
      "lat": 38.6287,
      "lon": -90.1988
    },
    {
      "lat": 37.3931,
      "lon": -121.962
    },
    {
      "lat": 37.3417,
      "lon": -121.9753
    },
    {
      "lat": 45.3151,
      "lon": -73.8779
    },
    {
      "lat": 1.35208,
      "lon": 103.82
    },
    {
      "lat": 40.8364,
      "lon": -74.1403
    },
    {
      "lat": 32.8137,
      "lon": -96.8704
    },
    {
      "lat": 38.6287,
      "lon": -90.1988
    },
    {
      "lat": 1.32123,
      "lon": 103.695
    },
    {
      "lat": -6.22636,
      "lon": 106.832
    },
    {
      "lat": 52.3891,
      "lon": 4.6563
    },
    {
      "lat": 47.6348,
      "lon": -122.3451
    },
    {
      "lat": 37.3931,
      "lon": -121.962
    },
    {
      "lat": 37.3931,
      "lon": -121.962
    },
    {
      "lat": 38.6287,
      "lon": -90.1988
    },
    {
      "lat": 50.7988,
      "lon": -1.0944
    },
    {
      "lat": 43.0351,
      "lon": -108.2024
    },
    {
      "lat": 34.0544,
      "lon": -118.2441
    },
    {
      "lat": 49.4521,
      "lon": 11.0767
    },
    {
      "lat": 50.7988,
      "lon": -1.0944
    },
    {
      "lat": 52.3716,
      "lon": 4.8883
    },
    {
      "lat": 41.8879,
      "lon": -88.1995
    },
    {
      "lat": 52.352,
      "lon": 4.9392
    },
    {
      "lat": 40.793,
      "lon": -74.0247
    },
    {
      "lat": 59.3293,
      "lon": 18.0686
    },
    {
      "lat": 40.7232,
      "lon": -74.0051
    },
    {
      "lat": 33.5939,
      "lon": -112.303
    },
    {
      "lat": 50.7988,
      "lon": -1.0944
    },
    {
      "lat": 60.1797,
      "lon": 24.9344
    },
    {
      "lat": 41.8879,
      "lon": -88.1995
    },
    {
      "lat": 49.4423,
      "lon": 11.0191
    },
    {
      "lat": 41.8879,
      "lon": -88.1995
    },
    {
      "lat": 50.1169,
      "lon": 8.6837
    },
    {
      "lat": 33.4475,
      "lon": -112.0866
    },
    {
      "lat": 50.1169,
      "lon": 8.6837
    },
    {
      "lat": 40.8364,
      "lon": -74.1403
    },
    {
      "lat": 1.32123,
      "lon": 103.695
    },
    {
      "lat": 38.6287,
      "lon": -90.1988
    },
    {
      "lat": 41.8879,
      "lon": -88.1995
    },
    {
      "lat": 40.973,
      "lon": -76.9117
    },
    {
      "lat": 32.7797,
      "lon": -96.8022
    },
    {
      "lat": 1.27989,
      "lon": 103.849
    },
    {
      "lat": 49.2402,
      "lon": 6.98519
    },
    {
      "lat": 33.5939,
      "lon": -112.303
    },
    {
      "lat": 60.1719,
      "lon": 24.9347
    },
    {
      "lat": 47.6034,
      "lon": -122.3414
    },
    {
      "lat": 33.5939,
      "lon": -112.303
    },
    {
      "lat": 52.5234,
      "lon": 13.3224
    },
    {
      "lat": 49.405,
      "lon": 11.1617
    },
    {
      "lat": 1.28223,
      "lon": 103.851
    },
    {
      "lat": 1.32123,
      "lon": 103.695
    },
    {
      "lat": 47.645,
      "lon": -117.0778
    },
    {
      "lat": 37.3931,
      "lon": -121.962
    },
    {
      "lat": 52.352,
      "lon": 4.9392
    },
    {
      "lat": 50.1169,
      "lon": 8.6837
    },
    {
      "lat": 30.3298,
      "lon": -81.6492
    },
    {
      "lat": 38.6287,
      "lon": -90.1988
    },
    {
      "lat": 40.793,
      "lon": -74.0247
    },
    {
      "lat": 52.352,
      "lon": 4.9392
    },
    {
      "lat": 38.6287,
      "lon": -90.1988
    },
    {
      "lat": 40.8364,
      "lon": -74.1403
    },
    {
      "lat": 40.793,
      "lon": -74.0247
    },
    {
      "lat": 47.9871,
      "lon": 17.0132
    },
    {
      "lat": 35.6092,
      "lon": 139.73
    },
    {
      "lat": 1.32123,
      "lon": 103.695
    },
    {
      "lat": 50.1169,
      "lon": 8.6837
    },
    {
      "lat": 40.793,
      "lon": -74.0247
    },
    {
      "lat": 37.7809,
      "lon": -122.4245
    },
    {
      "lat": 52.352,
      "lon": 4.9392
    },
    {
      "lat": 50.7988,
      "lon": -1.0944
    },
    {
      "lat": 47.6348,
      "lon": -122.3451
    },
    {
      "lat": 40.8364,
      "lon": -74.1403
    }
  ]
}
