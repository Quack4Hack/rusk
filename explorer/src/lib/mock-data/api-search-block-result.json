{
  "data": {
    "data": {
      "blocks": [
        {
          "header": {
            "hash": "fda46b4e06cc78542db9c780adbaee83a27fdf917de653e8ac34294cf924dd63"
          }
        }
      ]
    }
  }
}
