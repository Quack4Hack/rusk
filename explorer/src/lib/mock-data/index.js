export { default as apiMarketData } from "./api-market-data.json";
export { default as apiNodeLocations } from "./api-node-locations.json";
export { default as apiStats } from "./api-stats.json";
export { default as gqlBlock } from "./gql-block.json";
export { default as gqlBlocks } from "./gql-blocks.json";
export { default as gqlLatestChainInfo } from "./gql-latest-chain-info.json";
export { default as gqlSearchPossibleResults } from "./gql-search-possible-results.json";
export { default as gqlTransaction } from "./gql-transaction.json";
export { default as gqlTransactionDetails } from "./gql-transaction-details.json";
export { default as gqlTransactions } from "./gql-transactions.json";
