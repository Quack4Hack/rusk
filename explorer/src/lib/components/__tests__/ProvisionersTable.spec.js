import { afterEach, describe, expect, it } from "vitest";
import { cleanup, render } from "@testing-library/svelte";
import { slice } from "lamb";
import { hostProvisioners } from "$lib/mock-data";

import { ProvisionersTable } from "..";

describe("Provisioners Table", () => {
  const data = slice(hostProvisioners, 0, 10);

  const baseProps = {
    data: data,
  };
  const baseOptions = {
    props: baseProps,
    target: document.body,
  };

  afterEach(cleanup);

  it("should render the `ProvisionersTable` component", () => {
    const { container } = render(ProvisionersTable, baseOptions);

    expect(container.firstChild).toMatchSnapshot();
  });
});
