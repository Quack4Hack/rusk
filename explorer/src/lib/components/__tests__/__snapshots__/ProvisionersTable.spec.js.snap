// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Provisioners Table > should render the \`ProvisionersTable\` component 1`] = `
<div
  class="table-container provisioners-table"
>
  <table
    class="table"
  >
    <thead
      class="table__head"
    >
      <tr
        class="table__row"
      >
        <th
          class="table__header-cell"
        >
          Staking Address
        </th>
        
         
        <th
          class="table__header-cell"
        >
          Stake
        </th>
        
         
        <th
          class="table__header-cell"
        >
          Slashes
        </th>
        
         
        <th
          class="table__header-cell"
        >
          Accumulated Reward (DUSK)
        </th>
        
      </tr>
    </thead>
     
    <tbody
      class="table__body"
    >
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          m6dy2gz3jC...tW82FWtnpf
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          500,000
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          7,121.257296087
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          m9dVuRgr3C...94r33pFQyz
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          500,000
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          6,923.840604194
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mHGvQ9Xdjz...iTfxTvgn7f
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          4,800
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          4,320
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          89.505343279
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mLx5HUo5Ph...BrRgEzoka8
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          1,234
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          4,320
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          29.633646022
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mTgXDqkyVa...gAseybAsfU
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          500,000
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          7,518.756679536
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mUJHMDEBiT...FHNxRNhoAA
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          4,800
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          23,760
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          73.800637305
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mUUpjnXow2...SfzQcywzaW
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          500,000
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          7,653.541671166
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mWvq9EWD9b...oU7BY2oEDe
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          1,000
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          8,640
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          1.613840782
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mXKJ97xTtr...JgkCK2o7x4
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          500,000
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          7,512.060104659
        </td>
        
         
      </tr>
      
      <tr
        class="table__row"
      >
        <td
          class="table__data-cell"
        >
          mdXsA3Ee1Y...K8VZPimZ2J
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__stake-data-label"
          >
            Active:
          </b>
           
          500,000
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Inactive:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__stake-data-label"
          >
            Maturity At:
          </b>
          
          #
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          <b
            class="provisioners-table__slash-data-label"
          >
            Soft:
          </b>
           
          0
           
          <br />
           
          <b
            class="provisioners-table__slash-data-label"
          >
            Hard:
          </b>
           
          0
        </td>
        
         
        <td
          class="table__data-cell"
        >
          7,973.118109253
        </td>
        
         
      </tr>
      
    </tbody>
  </table>
</div>
`;
